.github/workflows/sage_test.yml

```diff
@@ -47,3 +47,15 @@ jobs:
 
       - name: Do the tests
         run: sage -python -m snappy.test
+
+      - name: Build docs
+        # Doesn't just do "make html" as neither "make" nor "zip" is
+        # installed in this image.
+        run: |
+          cd /__w/SnapPy/SnapPy/doc_src
+          sage -python -m sphinx -b html -E -d _build/doctrees . _build/doc
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: doc
+          path: /__w/SnapPy/SnapPy/doc_src/_build/doc/*
```

Comment:
It might be a mess to download all that stuff. Would it be hard to install zip in the container temporarily?

Reply:
The `upload-artifact` action automatically puts the files in a single zipfile for easy download.